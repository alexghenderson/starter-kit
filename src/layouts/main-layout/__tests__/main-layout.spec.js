import React from 'react';
import enzyme from 'enzyme';
import MainLayout from '../';

describe('Main Layout component', () => {
  it('should render without error', () => {
    const wrapper = enzyme.mount(<MainLayout>test-content</MainLayout>);
    expect(wrapper).toMatchSnapshot();
  });
});
