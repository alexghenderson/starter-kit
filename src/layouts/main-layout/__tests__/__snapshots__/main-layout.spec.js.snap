// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Layout component should render without error 1`] = `
Array [
  <ForwardRef(render)
    styles={
      Object {
        "map": "/*# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbInJlc2V0LmpzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQUllIiwiZmlsZSI6InJlc2V0LmpzIiwic291cmNlc0NvbnRlbnQiOlsiaW1wb3J0IHtHbG9iYWwsIGNzc30gZnJvbSAnQGVtb3Rpb24vY29yZSc7XG5cbmNvbnN0IHJlc2V0ID0gKFxuICA8R2xvYmFsXG4gICAgc3R5bGVzPXtjc3NgXG4gICAgICBodG1sLFxuICAgICAgYm9keSB7XG4gICAgICAgIGhlaWdodDogMTAwJTtcbiAgICAgIH1cbiAgICAgIGJvZHkge1xuICAgICAgICBkaXNwbGF5OiBmbGV4O1xuICAgICAgICBmbGV4LWRpcmVjdGlvbjogY29sdW1uO1xuICAgICAgfVxuICAgICAgYm9keSA+IG1haW4ge1xuICAgICAgICBmbGV4OiAxIDAgYXV0bztcbiAgICAgIH1cbiAgICAgIGh0bWwsXG4gICAgICBib2R5LFxuICAgICAgZGl2LFxuICAgICAgc3BhbixcbiAgICAgIGFwcGxldCxcbiAgICAgIG9iamVjdCxcbiAgICAgIGlmcmFtZSxcbiAgICAgIGgxLFxuICAgICAgaDIsXG4gICAgICBoMyxcbiAgICAgIGg0LFxuICAgICAgaDUsXG4gICAgICBoNixcbiAgICAgIHAsXG4gICAgICBibG9ja3F1b3RlLFxuICAgICAgcHJlLFxuICAgICAgYSxcbiAgICAgIGFiYnIsXG4gICAgICBhY3JvbnltLFxuICAgICAgYWRkcmVzcyxcbiAgICAgIGJpZyxcbiAgICAgIGNpdGUsXG4gICAgICBjb2RlLFxuICAgICAgZGVsLFxuICAgICAgZGZuLFxuICAgICAgZW0sXG4gICAgICBpbWcsXG4gICAgICBpbnMsXG4gICAgICBrYmQsXG4gICAgICBxLFxuICAgICAgcyxcbiAgICAgIHNhbXAsXG4gICAgICBzbWFsbCxcbiAgICAgIHN0cmlrZSxcbiAgICAgIHN0cm9uZyxcbiAgICAgIHN1YixcbiAgICAgIHN1cCxcbiAgICAgIHR0LFxuICAgICAgdmFyLFxuICAgICAgYixcbiAgICAgIHUsXG4gICAgICBpLFxuICAgICAgY2VudGVyLFxuICAgICAgZGwsXG4gICAgICBkdCxcbiAgICAgIGRkLFxuICAgICAgb2wsXG4gICAgICB1bCxcbiAgICAgIGxpLFxuICAgICAgZmllbGRzZXQsXG4gICAgICBmb3JtLFxuICAgICAgbGFiZWwsXG4gICAgICBsZWdlbmQsXG4gICAgICB0YWJsZSxcbiAgICAgIGNhcHRpb24sXG4gICAgICB0Ym9keSxcbiAgICAgIHRmb290LFxuICAgICAgdGhlYWQsXG4gICAgICB0cixcbiAgICAgIHRoLFxuICAgICAgdGQsXG4gICAgICBhcnRpY2xlLFxuICAgICAgYXNpZGUsXG4gICAgICBjYW52YXMsXG4gICAgICBkZXRhaWxzLFxuICAgICAgZW1iZWQsXG4gICAgICBmaWd1cmUsXG4gICAgICBmaWdjYXB0aW9uLFxuICAgICAgZm9vdGVyLFxuICAgICAgaGVhZGVyLFxuICAgICAgaGdyb3VwLFxuICAgICAgbWVudSxcbiAgICAgIG5hdixcbiAgICAgIG91dHB1dCxcbiAgICAgIHJ1YnksXG4gICAgICBzZWN0aW9uLFxuICAgICAgc3VtbWFyeSxcbiAgICAgIHRpbWUsXG4gICAgICBtYXJrLFxuICAgICAgYXVkaW8sXG4gICAgICB2aWRlbyB7XG4gICAgICAgIGJvcmRlcjogMDtcbiAgICAgICAgZm9udDogaW5oZXJpdDtcbiAgICAgICAgZm9udC1zaXplOiAxMDAlO1xuICAgICAgICBtYXJnaW46IDA7XG4gICAgICAgIHBhZGRpbmc6IDA7XG4gICAgICAgIHZlcnRpY2FsLWFsaWduOiBiYXNlbGluZTtcbiAgICAgIH1cblxuICAgICAgLyogSFRNTDUgZGlzcGxheS1yb2xlIHJlc2V0IGZvciBvbGRlciBicm93c2VycyAqL1xuICAgICAgYXJ0aWNsZSxcbiAgICAgIGFzaWRlLFxuICAgICAgZGV0YWlscyxcbiAgICAgIGZpZ2NhcHRpb24sXG4gICAgICBmaWd1cmUsXG4gICAgICBmb290ZXIsXG4gICAgICBoZWFkZXIsXG4gICAgICBoZ3JvdXAsXG4gICAgICBtZW51LFxuICAgICAgbmF2LFxuICAgICAgc2VjdGlvbiB7XG4gICAgICAgIGRpc3BsYXk6IGJsb2NrO1xuICAgICAgfVxuXG4gICAgICBib2R5IHtcbiAgICAgICAgbGluZS1oZWlnaHQ6IDE7XG4gICAgICB9XG5cbiAgICAgIG9sLFxuICAgICAgdWwge1xuICAgICAgICBsaXN0LXN0eWxlOiBub25lO1xuICAgICAgfVxuXG4gICAgICBibG9ja3F1b3RlLFxuICAgICAgcSB7XG4gICAgICAgIHF1b3Rlczogbm9uZTtcbiAgICAgIH1cblxuICAgICAgYmxvY2txdW90ZTo6YmVmb3JlLFxuICAgICAgYmxvY2txdW90ZTo6YWZ0ZXIsXG4gICAgICBxOjpiZWZvcmUsXG4gICAgICBxOjphZnRlciB7XG4gICAgICAgIGNvbnRlbnQ6ICcnO1xuICAgICAgfVxuXG4gICAgICB0YWJsZSB7XG4gICAgICAgIGJvcmRlci1jb2xsYXBzZTogY29sbGFwc2U7XG4gICAgICAgIGJvcmRlci1zcGFjaW5nOiAwO1xuICAgICAgfVxuXG4gICAgICBtYWluIHtcbiAgICAgICAgZGlzcGxheTogYmxvY2s7XG4gICAgICB9XG5cbiAgICAgICoge1xuICAgICAgICBiYWNrZ3JvdW5kLXJlcGVhdDogbm8tcmVwZWF0O1xuICAgICAgICBib3gtc2l6aW5nOiBib3JkZXItYm94O1xuXG4gICAgICAgIC1tb3otb3N4LWZvbnQtc21vb3RoaW5nOiBncmF5c2NhbGU7XG5cbiAgICAgICAgLXdlYmtpdC1mb250LXNtb290aGluZzogYW50aWFsaWFzZWQ7XG4gICAgICAgIHRleHQtcmVuZGVyaW5nOiBvcHRpbWl6ZUxlZ2liaWxpdHk7XG5cbiAgICAgICAgZm9udDogaW5oZXJpdDtcbiAgICAgICAgbGV0dGVyLXNwYWNpbmc6IC0wLjhweDtcblxuICAgICAgICAmOjpiZWZvcmUsXG4gICAgICAgICY6OmFmdGVyIHtcbiAgICAgICAgICBib3gtc2l6aW5nOiBib3JkZXItYm94O1xuICAgICAgICB9XG4gICAgICB9XG4gICAgYH1cbiAgLz5cbik7XG5cbmV4cG9ydCBkZWZhdWx0IHJlc2V0O1xuIl19 */",
        "name": "atgobn-main-layout-reset--reset",
        "styles": "html,body{height:100%;}body{display:flex;flex-direction:column;}body > main{flex:1 0 auto;}html,body,div,span,applet,object,iframe,h1,h2,h3,h4,h5,h6,p,blockquote,pre,a,abbr,acronym,address,big,cite,code,del,dfn,em,img,ins,kbd,q,s,samp,small,strike,strong,sub,sup,tt,var,b,u,i,center,dl,dt,dd,ol,ul,li,fieldset,form,label,legend,table,caption,tbody,tfoot,thead,tr,th,td,article,aside,canvas,details,embed,figure,figcaption,footer,header,hgroup,menu,nav,output,ruby,section,summary,time,mark,audio,video{border:0;font:inherit;font-size:100%;margin:0;padding:0;vertical-align:baseline;}article,aside,details,figcaption,figure,footer,header,hgroup,menu,nav,section{display:block;}body{line-height:1;}ol,ul{list-style:none;}blockquote,q{quotes:none;}blockquote::before,blockquote::after,q::before,q::after{content:'';}table{border-collapse:collapse;border-spacing:0;}main{display:block;}*{background-repeat:no-repeat;box-sizing:border-box;-moz-osx-font-smoothing:grayscale;-webkit-font-smoothing:antialiased;text-rendering:optimizeLegibility;font:inherit;letter-spacing:-0.8px;&::before,&::after{box-sizing:border-box;}}label:main-layout-reset--reset;",
      }
    }
  />,
  "test-content",
]
`;
